lib/metrics.js

```diff
@@ -30,6 +30,12 @@ exports.startMonitoring = (
         labelNames: ['path', 'method', 'status'],
         percentiles: [0.1, 0.5, 0.9, 0.99],
     });
+    const dbDuration = new client.Summary({
+        name: 'db_query_duration_milliseconds',
```

Comment:
Metrics should be base units (i.e. seconds in this case)

Reply:
Good point, i'll change that!
There's something to rename in 4.x :) [http_request_duration_milliseconds](https://github.com/Unleash/unleash/blob/master/lib/metrics.js#L28)